src/doom/st_stuff.c

```diff
@@ -1490,14 +1497,13 @@ void ST_doPaletteStuff(void)
     {
         palette = RADIATIONPAL;
     }
-
+    }
```

Comment:
This is a good idea, thanks for the PR!

However, the `else` part of your patch nearly spans the entire previous function. I would rather keep it as it is and just override the `palette` value right before this line.

Reply:
I implemented it as such because if it turns out the player is in the help screen or crispness menu, it skips having to go through all of those other conditions that determine palette changes. But if you still prefer it be done as you said, I can change that.

Reply:
Yes, indeed I prefer patches to touch the original code as little as possible. Sure, the engine will have to do some unnecessary calculations, but it will be clearer what the patch is supposed to do if it doesn't span across the entire function. It may be even so clear that the compiler optimizes in a way that is similar to your original approach. ;)

Oh, and please add a short one-line comment behind the `[crispy]` marker.